src/components/FilterBar/FilterDisplay/Inputs/DateInput.react.js

```diff
@@ -2,10 +2,16 @@ export class DateInput extends React.Component {
   constructor(props) {
     super(props);
 
-    this.state = { value: this.props.value || { from: null, to: null } };
+    this.state = {
+      value: this.props.value || { from: null, to: null },
+      displayFrom: this.props.displayFrom,
```

Comment:
Allows us to display dates passed in from the RelativeDateInput without storing them in the filter value.